src/pages/results/results.page.tsx

```diff
@@ -1,12 +1,77 @@
-import React from 'react';
-import './results.scss';
+import React, { useEffect, useState } from 'react';
+import { Select } from 'antd';
+import Abstract from "../../components/abstract/abstract";
 
-interface IProps { }
 
-const Results = (props: IProps) => {
-    return (
-        <div className='results'>Results</div>
-    )
-}
+const { Option } = Select;
+const [loading] = useState(false);
+
+
+TODO: once the back-end is ready this comment code can be used to fetch the whole tages from the database.
+// const Results = () => {
+//   const [options, setOptions] = useState([]);
+//   const [loading, setLoading] = useState(true);
+//   const [searchArray, setSearchArray] = useState([]);
+
+//   useEffect(() => {
+//     fetch('/tags')
+//       .then(response => response.json())
+//       .then(data => {
+//         setOptions(data);
+//         setLoading(false);
+//       });
+//   }, []);
+
+
+const Results = () => {
+  const [options] = useState([
+    { value: 'Ai', label: 'Ai' },
+    { value: 'Math', label: 'Math' },
+    { value: 'Chem', label: 'Chem' },
+  ]);
+
+  const [searchArray, setSearchArray] = useState([
+    { abstract: 'the first abstract', tag: 'Ai' },
+    { abstract: 'the second abstract', tag: 'Chem' },
+  ]);
+
+  const handleSearch = value => {
```

Comment:
This need a type annotation, are you sure that you have implicit `any` disabled?

Reply:
done, after you read the code.